src/mixins/graphql.js

```diff
@@ -51,7 +52,8 @@ export default {
      * @return {string} - the Workflow ID used in this view
      */
     workflowId () {
-      return `${this.user.username}|${this.workflowName}`
+      const username = getUserNameFromUrl(this.user.username)
+      return `${username}|${this.workflowName}`
```

Comment:
The Vuex user is still there, but we will try to use the user found in the URL. If no user found, then we fallback to the Vuex value.